ports/esp32/boards/UM_FEATHERS3/board.json

```diff
@@ -0,0 +1,27 @@
+{
+    "deploy": [
+        "deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "Battery Charging",
+        "RGB LED",
+        "SPIRAM",
+        "USB-C",
+        "WiFi",
+        "BLE",
+        "STEMMA QT/QWIIC",
+        "Feather"
+    ],
+    "features_non_filterable": [
+    ],
+    "id": "feathers3",
+    "images": [
+        "photo_feathers3.jpg"
```

Comment:
This doesn't match the image name at https://github.com/micropython/micropython-media/pull/10.  They must match (eg change this here to `unexpectedmaker_feathers3.jpg`).

Also the other 2 images in this PR need fixing.

Reply:
Those image names are now fixed, and I've added the MIT license message + cleaned up the ADC read code in the get_battery_voltage() methods.

Reply:
I see you're still using `.read()`, which is fine.  But out of interest, what is the circuit used to measure the battery voltage?  Will the input on this pin ever be > 1v?  If so you may want to use attenuation on this ADC pin, to avoid damage to the ESP32.

Reply:
It's 1.1V at 4.2V - Same as I have always had on my TinyPICO, and S2 boards.  Never seen an issue at this voltage, but that doesn't mean it's good for the ESP32 - or bad.

Reply:
Ok.